source/blog/2018-09-07-the-ember-times-issue-63.md

```diff
@@ -54,8 +54,27 @@ Read either on the [Ember blog](https://emberjs.com/blog/tags/newsletter.html) o
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [A bigger Ember tent ⛺](https://discuss.emberjs.com/t/a-bigger-ember-tent/15383)
+
+The expression 'Ember is a big tent' was recently used by Tom Dale in response to an Ember developer when he asked if [Ember was still a welcoming community](https://discuss.emberjs.com/t/are-we-still-a-welcoming-community/15285).
```

Comment:
Can we link Tom's github https://github.com/tomdale

Reply:
Of course! :) thank you.